src/cpp/rtps/xmlparser/XMLParser.cpp

```diff
@@ -710,6 +710,14 @@ XMLP_ret XMLParser::parseXMLCommonTCPTransportData(
                     return XMLP_ret::XML_ERROR;
                 }
             }
+            // non_blocking_send - boolType
```

Comment:
We should update [this](https://github.com/eProsima/Fast-DDS/blob/37b0c4892b7f6f661aaf1b4cfebefe8ab47470bf/resources/xsd/fastRTPS_profiles.xsd#L856), changing the comment to read `NOT  available for   SHM type`